### PR TITLE
INFRA : Initialize Unit Test

### PR DESCRIPTION
**Pull request by GracelinJenifa**

### CODE REVIEW

File: AddressApp.Tests.Unit/DeleteMe.cs [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+using Xunit;
+
+namespace AddressApp.Tests.Unit
+{
+    public class DeleteMe
+    {
+        [Fact]
+        public void ShouldBeTrue() =>
+        Assert.True(true);
```
Comment (ShriHumrudha) [inline AddressApp.Tests.Unit/DeleteMe.cs]: Correct the indentation here.

**Review by ShriHumrudha — CHANGES_REQUESTED** [review]

File: AddressApp.Tests.Unit/DeleteMe.cs [hunk 2]
```diff
@@ -8,8 +8,8 @@ namespace AddressApp.Tests.Unit
 {
     public class DeleteMe
     {
-        [Fact]
+        [Fact]
```
Comment (ShriHumrudha) [inline AddressApp.Tests.Unit/DeleteMe.cs]: Please run code cleanup to remove unwanted spaces.

**Review by ShriHumrudha — CHANGES_REQUESTED** [review]

File: AddressApp.Tests.Unit/DeleteMe.cs (hunk 2 above)
Comment (GracelinJenifa) [reply AddressApp.Tests.Unit/DeleteMe.cs]: @ShriHumrudha  I have run the code cleanup. Please check and let me know your feedback.

**Review by ShriHumrudha — APPROVED** [review]

**Comment by ShriHumrudha** [comment]
@GracelinJenifa You need to resolve the converstaions.

**Pull request merged**